runtime/src/group.rs

```diff
@@ -101,7 +101,7 @@ decl_module! {
         /// * `origin` Origin representing `AddOrigin` or root
         /// * `who` IdentityId to be added to the group.
         #[weight = SimpleDispatchInfo::FixedNormal(50_000)]
-        fn add_member(origin, who: IdentityId) {
+        pub fn add_member(origin, who: IdentityId) {
```

Comment:
Making this public isn't a good idea. We use an instance of `group` in another module so as to control who can add/remove members through governance for example. Please see below for testing suggestions.